tests/cloudpickle_test.py

```diff
@@ -2170,22 +2093,13 @@ def check_generic(generic, origin, type_value):
             assert check_generic(C[int], C, int) == "ok"
             assert worker.run(check_generic, C[int], C, int) == "ok"
 
-    @unittest.skipIf(sys.version_info < (3, 7),
-                     "Pickling type hints not supported below py37")
     def test_locally_defined_class_with_type_hints(self):
```

Comment:
We could, but it seemed worth it to do this change because then it tests pickling/unpickling class attributes on 3.5 too. That could happen in real code if a pickle created from 3.6+ is loaded.

Another small reason was that assigning `__annotations__` gets rid of the need to parse the code dynamically.

Reply:
> That could happen in real code if a pickle created from 3.6+ is loaded.

FYI, neither `pickle` nor `cloudpickle`  supports pickling an object using `Python 3.6` and depickling it using `Python 3.5` :)

> Another small reason was that assigning __annotations__ gets rid of the need to parse the code dynamically.

I agree. But the current syntax makes this test somewhat obscure, for future maintenance I would prefer using the `object: TypeHint` syntax explicitly.